frontend/javascripts/oxalis/view/action-bar/python_client_modal_view.tsx

```diff
@@ -0,0 +1,107 @@
+import { Divider, Modal, Row, Typography } from "antd";
+import React from "react";
+import { useFetch } from "libs/react_helpers";
+import Toast from "libs/toast";
+import messages from "messages";
+import { getAuthToken } from "admin/admin_rest_api";
+import { useSelector } from "react-redux";
+import type { OxalisState } from "oxalis/store";
+import { CopyableCodeSnippet, MoreInfoHint } from "./download_modal_view";
+const { Paragraph, Text } = Typography;
+type Props = {
+  onClose: () => void;
+};
+
+export default function PythonClientModalView(props: Props): JSX.Element {
```

Comment:
In a recent commit, I changed the modals to become lazy so that they are only rendered after they were opened at least once. You can have a look at how this can be done by searching for `const DownloadModalView = makeComponentLazy(_DownloadModalView);` and adopting that.

Reply:
ah, yeah, good point :+1: apparently, I missed the dataset-view-mode related modals in my PR. thank you!